docs/troubleshoot/troubleshoot.md

```diff
@@ -0,0 +1,237 @@
+# Percona Everest: Troubleshooting and diagnostics
+
+This section is your go-to resource for tackling common issues and finding solutions. 
+
+For known issues, see the [known limitations](../reference/known_limitations.md) section.
+
+
+## General troubleshooting guidelines
+
+Before troubleshooting, it's important to understand how Percona Everest works at a high level. Understanding how it works can help narrow down potential areas to investigate.
+
+- Refer to the [kubectl quick reference](https://kubernetes.io/docs/reference/kubectl/quick-reference/){:target="_blank"} for commonly used commands to inspect and troubleshoot Kubernetes resources when working with Percona Everest.
+
+- Use logs and events for debugging.
+
+
+### Key logs and commands
+
+You can review different logs for additional information depending on the specific issue.
+{.power-number}
+
+1. **Everest Core Components**
+
+    | Logs        | Command                                                     |
+    | ---------------- | ----------------------------------------------------------- |
+    | Percona Everest operator| `kubectl logs -f deploy/everest-operator -n everest-system` |
+    | Percona Everest server | `kubectl logs -f deploy/everest-server -n everest-system`   |
+
+2. **Database Operators (in Namespaces)**
+
+    | Operator type | Namespace | Command                                                             |
+    | ------------- | --------- | ------------------------------------------------------------------- |
+    | PostgreSQL    | `everest` | `kubectl logs -f deploy/percona-postgresql-operator -n everest`     |
+    | MongoDB       | `everest` | `kubectl logs -f deploy/percona-server-mongodb-operator -n everest` |
+    | PXC           | `everest` | `kubectl logs -f deploy/percona-xtradb-cluster-operator -n everest` |
+
+    !!! note
+        Update the namespace (`-n <namespace>`) in your commands if your database clusters are deployed in a namespace other than the default one.
+
+3. **Monitoring**
+
+    | Component                | Command                                                                   |
+    | ------------------------ | ------------------------------------------------------------------------- |
+    | VictoriaMetrics Operator | `kubectl logs -f deploy/vm-operator -n everest-monitoring`                |
+    | VM Agent                 | `kubectl logs -f deploy/vmagent-everest-monitoring -n everest-monitoring` |
+
+
+4. **Database and Proxy Pods**
+
+    ```sh
+    kubectl logs -f <pod-name of database or proxy> -c <database-container>
+    ```
+
+5. **Kubernetes Events**
+
+    !!! info "Important"
+        Events (Events are stored only for 60 mins; if there are any events older than 60 minutes, they will be lost).
+
+    ```sh
+    kubectl get events --sort-by=".lastTimestamp"
+    ```
+
+## Troubleshooting key areas
+
+### Installation issues
```

Comment:
This section should be with the Installation instructions. I wouldn't know that I should scroll down to find out what to do.